Source/Core/GeometryPipeline.js

```diff
@@ -2552,7 +2552,7 @@ define([
         var boundingSphere = geometry.boundingSphere;
         if (defined(boundingSphere)) {
             var minX = boundingSphere.center.x - boundingSphere.radius;
-            if (minX > 0 || BoundingSphere.intersectPlane(boundingSphere, Plane.ORIGIN_ZX_PLANE) !== Intersect.INTERSECTING) {
+            if (minX > 0 || BoundingSphere.intersectPlane(boundingSphere, Plane.ORIGIN_ZX_PLANE) !== Intersect.INTERSECTING || boundingSphere.center.equals(Cartesian3.ZERO)) {
```

Comment:
It would hide the problem for some. A large rectangle that covers the surface of the entire Earth would be fine, but a box geometry without a model matrix to move it from the origin cannot be projected to 2D.